src/pcre2_compile.c

```diff
@@ -6388,6 +6388,12 @@ for (;; pptr++)
         range = end;
         }
 
+#if PCRE2_CODE_UNIT_WIDTH == 8
+      /* If code unit width is 8 bits, and UCP flag is set, but UTF flag is not, we still
+       * generate cranges, but in that case we should not process any crange > 0xFF,
+       * because it's impossible to encounter code points > 0xFF in the subject string */
+      if (utf)
```

Comment:
Do we need code indentation below? An `if (!utf) range = end;` also works. I have no preference.

Reply:
Avoiding conditional indentation can only help making the logic easier to follow IMHO